ui/src/components/Publication/RelatedPublications/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import * as Components from '@/components';
+import * as Interfaces from '@/interfaces';
+
+type Props = {
+    id: string;
+    crosslinks: Interfaces.MixedCrosslinks;
+};
+
+const RelatedPublications: React.FC<Props> = (props) => {
+    const { recent, relevant } = props.crosslinks;
+    return recent.length || relevant.length ? (
+        <Components.AccordionSection id={props.id} title={'Related Publications'}>
+            <div className="space-y-4 px-6 py-4">
+                {!!recent.length && (
+                    <section className="flex flex-col">
+                        {!!relevant.length && (
+                            <span className="uppercase leading-0 block font-montserrat text-xs font-bold tracking-wide text-teal-400 dark:text-teal-200">
+                                Most recent
+                            </span>
+                        )}
+                        {props.crosslinks.recent.map((crosslink) => (
+                            <Components.RelatedPublicationsCard
+                                crosslink={crosslink}
+                                key={crosslink.linkedPublication.id}
+                            />
+                        ))}
+                    </section>
+                )}
+                {!!relevant.length && (
```

Comment:
Curious, you double bang `relevant.length` here, but not above on line 12

Reply:
Yes, I think if it precedes a && like on line 30, it will still render a "0" if there is no length, unless you add "!!". But because line 12 is using a ternary operator instead of "&&", it won't render the result of the initial check in the event that it's falsy.